scripts/releaseV2/phase2-bump-all-packages.mjs

```diff
@@ -0,0 +1,80 @@
+import {
+  getLastTag,
+  parseCommits,
+  getCommits,
+  angular,
+  npmBumpVersion,
+  npmPublish,
+  generateChangelog,
+  writeChangelog,
+} from '@coveo/semantic-monorepo-tools';
+import {spawnSync} from 'child_process';
+import {readFileSync} from 'fs';
+
+// Run on each package, it generate the changelog, install the latest dependencies that are part of the workspace, publish the package.
+(async () => {
+  const PATH = '.';
+  const versionPrefix = 'v';
+  const awaitedangular = await angular;
```

Comment:
I think the word `angular` is misleading. I though it has to do with the Angular template.

Reply:
Fixed, replaced by 'convention'.
Reworked the separation of concern, the user of the tools should provide its own conventional thingy